src/styles/menu-slider.js

```diff
@@ -1,5 +1,7 @@
 import { css } from 'lit-element';
 
+const menuButtonWidth = css`42px`;
```

Comment:
Should this be configurable via a CSS Var?

Reply:
Not as far as we're going to use it. It's a variable to keep both uses of this measurement in sync.